static/app/views/dashboardsV2/dashboard.tsx

```diff
@@ -1,10 +1,17 @@
+import 'react-grid-layout/css/styles.css';
```

Comment:
Wondering if there's a pattern we can use to only import these if the flag is set 🤔

Reply:
Going to punt on this. I tried adding the import step to `componentWillMount` where I can check the flag, but I was getting TS errors saying the module, or its types, couldn't be found. At this point I figured trying to lazy load will overcomplicate things